Docs/source/usage/parameters.rst

```diff
@@ -1913,6 +1913,22 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz``.
     Note that the fields are averaged on the cell centers before they are written to file.
 
+* ``<diag_name>.particle_fields_to_plot`` (list of `strings`, optional)
+   Names of per-cell averages of particle properties to calculate and output. Default is none. Parser functions for these field names are specified by ``particle_fields.<field_name>(x,y,z,ux,uy,uz)``.
```

Comment:
For preciseness, this should be `<diag_name>.particle_fields.<field_name>(x,y,z,ux,uy,uz)`, including the `<diag_name>` prefix.